presto-spark/src/main/java/com/facebook/presto/spark/SparkTaskDescriptor.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark;
+
+import com.facebook.presto.SessionRepresentation;
+import com.facebook.presto.execution.TaskSource;
+import com.facebook.presto.execution.scheduler.TableWriteInfo;
+import com.facebook.presto.sql.planner.PlanFragment;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+public class SparkTaskDescriptor
```

Comment:
nit: `PrestoOnSparkTaskDescriptor` or `PrestoOnSparkTaskRequest`?

Reply:
It is technically not a request. Let's keep it as Descriptor.